tests/lib/prepare-utils.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
 set -e
-set -x 
+set -x
 
 SSH_PORT=${SSH_PORT:-8022}
 MON_PORT=${MON_PORT:-8888}
+# For os.query and others
+PATH=$PATH:$PROJECT_PATH/tests/lib/external/snapd-testing-tools/tools/
```

Comment:
you can also either move binaries to the path in spread.yaml or to include this in the nevironment in spread.yaml.

Reply:
Good point, changed now